Lib/test/test_builtin.py

```diff
@@ -2494,18 +2498,27 @@ def test_type_name(self):
             A.__name__ = b'A'
         self.assertEqual(A.__name__, 'C')
 
+        # if __module__ is not a string, ignore it silently
+        class D:
+            pass
+        self.assertEqual(D.__fully_qualified_name__, f'{__name__}.{D.__qualname__}')
+        D.__module__ = 123
+        self.assertEqual(D.__fully_qualified_name__, D.__qualname__)
```

Comment:
Can you test the `__fully_qualified_name__` attribute on some builtin classes?

Reply:
Sure, done.